package.json

```diff
@@ -109,22 +109,23 @@
     "@babel/generator": "^7.0.0",
     "@reactions/component": "^2.0.2",
     "async": "^2.4.0",
-    "babel-eslint": "9.0.0",
+    "babel-eslint": "10.0.1",
     "clang-format": "^1.2.4",
     "coveralls": "^3.0.2",
     "detox": "9.0.4",
-    "eslint": "5.1.0",
-    "eslint-config-fb-strict": "22.1.0",
-    "eslint-config-fbjs": "2.0.1",
-    "eslint-plugin-babel": "5.1.0",
-    "eslint-plugin-eslint-comments": "^3.0.1",
-    "eslint-plugin-flowtype": "2.43.0",
-    "eslint-plugin-jest": "21.8.0",
-    "eslint-plugin-jsx-a11y": "6.0.3",
-    "eslint-plugin-prettier": "2.6.0",
-    "eslint-plugin-react": "7.8.2",
-    "eslint-plugin-react-hooks": "^1.0.1",
-    "eslint-plugin-react-native": "3.5.0",
+    "eslint": "5.15.2",
+    "eslint-config-fb-strict": "24.3.0",
+    "eslint-config-fbjs": "2.1.0",
+    "eslint-plugin-babel": "^5.3.0",
+    "eslint-plugin-eslint-comments": "^3.1.1",
+    "eslint-plugin-flowtype": "2.50.3",
+    "eslint-plugin-jest": "22.4.1",
+    "eslint-plugin-jsx-a11y": "6.2.1",
```

Comment:
Looks like this plugin is targeting web: https://github.com/evcohen/eslint-plugin-jsx-a11y#supported-rules
It's also not enabled in the config's plugin list. I think that can be removed.

For RN there's https://github.com/FormidableLabs/eslint-plugin-react-native-a11y but it stopped receiving support shortly after release and iirc it still lacks some accessibility features introduced in 0.57